UnitTests/BraintreeSEPADirectDebitTests/BTSEPADirectDebitClient_Tests.swift

```diff
@@ -0,0 +1,7 @@
+import XCTest
```

Comment:
Do we need to add this test class before we've added any XCTestCases?

Reply:
I think this is fine for now as this is all going into a feature branch and we will add tests as we add implementation pieces and will for sure add client tests.